ngraph/python/tests/test_onnx/model_zoo_preprocess.sh

```diff
@@ -36,37 +36,44 @@ done
 shift $((OPTIND -1))
 
 if [ $MODELS_DIR = false ] ; then
-	echo "Unknown location of the ONNX ZOO models"
+	echo "Unknown location of the ZOO models"
 	exit 170
 fi
+ONNX_MODELS_DIR=$MODELS_DIR/model_zoo/onnx_model_zoo
+MSFT_MODELS_DIR=$MODELS_DIR/model_zoo/MSFT
 
 if [ $CLONE = true ] ; then
 	if [ $CLEAN_DIR = true ] ; then
 		rm -rf $MODELS_DIR
 	fi
-	git clone https://github.com/onnx/models.git $MODELS_DIR
+	git clone https://github.com/onnx/models.git $ONNX_MODELS_DIR
 fi
 
-cd $MODELS_DIR
+mkdir -p $ONNX_MODELS_DIR $MSFT_MODELS_DIR
+cd $ONNX_MODELS_DIR
 # remove already downloaded models
 git clean -f -x -d
 git checkout .
 git pull -p
 # pull models from the lfs repository
 # onnx models are included in the tar.gz archives
 git lfs pull --include="*" --exclude="*.onnx"
-find $MODELS_DIR -name "*.onnx" | while read filename; do rm "$filename"; done;
+find $ONNX_MODELS_DIR -name "*.onnx" | while read filename; do rm "$filename"; done;
 echo "extracting tar.gz archives..."
-find $MODELS_DIR -name '*.tar.gz' -execdir sh -c 'BASEDIR=$(basename "{}" .tar.gz) && mkdir -p $BASEDIR' \; -execdir sh -c 'BASEDIR=$(basename "{}" .tar.gz) && tar -xzvf "{}" -C $BASEDIR --strip-components=1' \;
+find $ONNX_MODELS_DIR -name '*.tar.gz' -execdir sh -c 'BASEDIR=$(basename "{}" .tar.gz) && mkdir -p $BASEDIR' \; -execdir sh -c 'BASEDIR=$(basename "{}" .tar.gz) && tar -xzvf "{}" -C $BASEDIR' \;
 # fix yolo v4 model
-cd $MODELS_DIR/vision/object_detection_segmentation/yolov4/model/yolov4/yolov4/test_data_set
+cd $ONNX_MODELS_DIR/vision/object_detection_segmentation/yolov4/model/yolov4/yolov4/test_data_set
 mv input0.pb input_0.pb
 mv input1.pb input_1.pb
 mv input2.pb input_2.pb
 mv output0.pb output_0.pb
 mv output1.pb output_1.pb
 mv output2.pb output_2.pb
 # fix roberta model
-cd $MODELS_DIR/text/machine_comprehension/roberta/model/roberta-sequence-classification-9/
+cd $ONNX_MODELS_DIR/text/machine_comprehension/roberta/model/roberta-sequence-classification-9/roberta-sequence-classification-9
 mkdir test_data_set_0
 mv *.pb test_data_set_0/
+
+# Prepare MSFT models
+wget -O $MSFT_MODELS_DIR/20191107.zip https://onnxruntimetestdata.blob.core.windows.net/models/20191107.zip
+unzip $MSFT_MODELS_DIR/20191107.zip -d $MSFT_MODELS_DIR/20191107 && rm $MSFT_MODELS_DIR/20191107.zip
```

Comment:
this file is pretty big so maybe its worth to add an option to skip(or enable) download of it?

Reply:
by default it was set as disabled - to download this special flag should be used